src/content/docs/pt-br/reference/configuration-reference.mdx

```diff
@@ -503,60 +508,81 @@ de configuração especiais para redirecionamentos e não precisam/querem redire
- - `'auto'` - apenas folhas de estilo menores que `ViteConfig.build.assetsInlineLimit` (padrão: 4kb) são embutidas. Caso contrário, estilos do projeto são enviados em folhas de estilo externas.
- - `'never'` - estilos do projeto são enviados em folhas de estilo externas
+  - `'always'` - estilos do projeto são embutidos em tags `<style>`
+  - `'auto'`   - apenas folhas de estilo menores que `ViteConfig.build.assetsInlineLimit` (padrão: 4kb) são embutidas. Caso contrário, estilos do projeto são enviados em folhas de estilo externas.
+  - `'never'`  - estilos do projeto são enviados em folhas de estilo externas
 
 ```js
 {
-	build: {
-		inlineStylesheets: `never`,
-	},
+  build: {
+    inlineStylesheets: `never`,
+  },
 }
 ```
 
-
-### build.split
+## Opções de pré-carregamento
 
 <p>
 
-**Tipo:** `boolean`<br />
-**Padrão:** `false`
+**Tipo:** `boolean | object`
 </p>
 
-A opção de configuração da build `build.split` foi substituída pela opção de configuração do adaptador [`functionPerRoute`](/pt-br/reference/adapter-reference/#functionperroute).
+Habilita o pré-carregamento de links no seu site para oferecer transições de páginas mais rápidas.
+(Habilitado por padrão em páginas que usam o roteador `<ViewTransitions />`. Defina `prefetch: false` para optar por não ter esse comportamento.)
 
-Por favor veja a [documentação do seu adaptador SSR](/pt-br/guides/integrations-guide/#integrações-oficiais) para utilizar `functionPerRoute` para definir como seu código SSR deve passar por bundle.
+Essa configuração adiciona um script de pré-carregamento automaticamente em todas as páginas no projeto
+lhe dando acesso ao atributo `data-astro-prefetch`.
+Adicione esse atributo em qualquer link `<a />` em sua página para habilitar o pré-carregamento para aquela página.
 
-```js
-{
-  build: {
-    split: true
-  }
-}
+```html
+<a href="/sobre" data-astro-prefetch>Sobre</a>
 ```
+Customize mais o comportamente padrão de pré-carregamento utilizando as opções [`prefetch.defaultStrategy`](#prefetchdefaultstrategy) e [`prefetch.prefetchAll`](#prefetchprefetchall).
 
+Veja o [guia de Pré-carregamento](/pt-br/guides/prefetch/) para mais informações.
 
-### build.excludeMiddleware
+### prefetch.prefetchAll
 
 <p>
 
-**Tipo:** `boolean`<br />
-**Padrão:** `false`
+**Tipo:** `boolean`
 </p>
 
-A opção de configuração da build `build.excludeMiddleware` foi substituída pela opção de configuração do adaptador [`edgeMiddleware`](/pt-br/reference/adapter-reference/#edgemiddleware).
-
-Por favor veja a [documentação do seu adaptador SSR](/pt-br/guides/integrations-guide/#integrações-oficiais) para utilizar `edgeMiddleware` para definir se algum ou nenhum código de middleware SSR irá passar por bundle durante a build.
+Habilita o pré-carregamento para todos os links, incluindo aqueles sem o atributo `data-astro-prefetch`.
+Esse valor padrão é `true` quando utilizando o roteador `<ViewTransitions />`. Caso contrário, o valor padrão é `false`.
```

Comment:
```suggestion
Este valor é `true` por padrão quando utilizando o roteador `<ViewTransitions />`. Caso contrário, o valor padrão é `false`.
```

Reply:
"Este" is for something that will be introduced in the text.
"Esse" is for something already introduced in the text.

It is referring to the value described in the previous line so "esse" is correct.